client/src/app/services/theme.service.ts.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@angular/core';
```

Comment:
Rename this file to `theme.service.ts`

Reply:
Fixed in: https://github.com/CMEPW/Smersh/pull/49/commits/69d0cef7dab090e012dcf0784c237b7c1797cb18